src/mesh/voxel_cloud.rs

```diff
@@ -538,6 +538,36 @@ impl ScalarField {
     ) where
         U: RangeBounds<f32>,
     {
+        use std::ops::Bound::*;
+
+        // FIXME: This discards the `other` scalar field's fine-grain value
```

Comment:
T_T We'll have to think about this in a more general sense. Thanks for documenting it in the fixme.

Reply:
I believe this is a philosophical question - what is the validity of a scalar field when exposed to another, in general? Should we take into consideration only the volume value range or also the values around it? What about infinite (very common cases) volume value ranges?
This fix should be enough for the UI operations we have, though.